src/main/scala/DocGenerator.scala

```diff
@@ -39,13 +38,18 @@ object PlainStringGenerator extends DocGenerator {
 object LatexDocGenerator extends DocGenerator {
 
 
+  def generateIndex(index: Index): String = {
+    "\\newpage\n" +
+      "Generated INDEX" + indexForObjects(index) + "\n" +
+      "Generated INDEX" + indexForMethods(index) + "\n" +
+      "Generated INDEX" + indexForClasses(index) + "\n" +
+      "Generated INDEX" + indexForTraits(index) + "\n"
+  }
```

Comment:
I'm not sure that this will produce good output. The X in INDEX and the return value of indexForX will be right up against each other with no spaces. Also, why aren't we using a triple-quoted String?